src/components/Header.js

```diff
@@ -1,12 +1,60 @@
+import {Drawer} from '@mui/material';
+import Image from 'next/image';
 import Link from 'next/link';
+import {useState} from 'react';
+import {BsFillPeopleFill} from 'react-icons/bs';
+import * as FaIcons from 'react-icons/fa';
+import {MdHomeRepairService} from 'react-icons/md';
```

Comment:
I would recommend using uniform icons from on set. e.g. Material Design icons. But that is more a Design Question 💅